code/modules/events/holiday/xmas.dm

```diff
@@ -37,7 +37,15 @@
 	icon_state = "xmashat"
 	desc = "A crappy paper hat that you are REQUIRED to wear."
 	flags_inv = 0
-	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0, "stamina" = 0)
+
+/obj/item/clothing/head/festive/Initialize()
+	. = ..()
+	//Merry christmas
+	if(CHRISTMAS in SSevents.holidays)
+		armor = list("melee" = 30, "bullet" = 30, "laser" = 30,"energy" = 30, "bomb" = 30, "bio" = 30, "rad" = 30, "fire" = 30, "acid" = 30, "stamina" = 30)
+	else if(FESTIVE_SEASON in SSevents.holidays)
+		armor = list("melee" = 20, "bullet" = 20, "laser" = 20,"energy" = 20, "bomb" = 20, "bio" = 20, "rad" = 20, "fire" = 20, "acid" = 20, "stamina" = 20)
```

Comment:
You actually have to initialize an actual armor at this point, since armor datum is initialized at `/obj/Initialize`. Therefore, set this before the parent call:

```suggestion
/obj/item/clothing/head/festive/Initialize()
	//Merry christmas
	if(CHRISTMAS in SSevents.holidays)
		armor = list("melee" = 30, "bullet" = 30, "laser" = 30,"energy" = 30, "bomb" = 30, "bio" = 30, "rad" = 30, "fire" = 30, "acid" = 30, "stamina" = 30)
	else if(FESTIVE_SEASON in SSevents.holidays)
		armor = list("melee" = 20, "bullet" = 20, "laser" = 20,"energy" = 20, "bomb" = 20, "bio" = 20, "rad" = 20, "fire" = 20, "acid" = 20, "stamina" = 20)
	return ..()
```